.pre-commit-config.yaml

```diff
@@ -14,13 +14,24 @@ repos:
   - id: debug-statements
   - id: forbid-new-submodules
 - repo: https://github.com/psf/black
-  rev: stable
+  rev: 19.10b0
   hooks:
   - id: black
 - repo: https://gitlab.com/pycqa/flake8
-  rev: 3.8.1
+  rev: "3.8.1"
   hooks:
   - id: flake8
+    additional_dependencies: ["flake8-bugbear == 20.1.2"]
+    language_version: python3.8
+- repo: https://github.com/asottile/seed-isort-config
+  rev: v2.1.1
+  hooks:
+  - id: seed-isort-config
+    args: [--application-directories, '.:src']
+- repo: https://github.com/asottile/setup-cfg-fmt
```

Comment:
add formatter for setup.cfg :-) similar as black is for python files. Bonus it automatically updates the python classifiers based on python requires, to ensure they are always in sync

Reply:
I'll take a look at this and see what I think (see below).